src/nanoemoji/config.py

```diff
@@ -297,8 +298,8 @@ def _resolve_config(
     if config_file is None:
         with resources.path("nanoemoji.data", _DEFAULT_CONFIG_FILE) as config_file:
             # no config_dir in this context; bad input if we need it
-            return None, toml.load(config_file)
-    return config_file.parent, toml.load(config_file)
+            return None, tomlkit.load(open(config_file))
```

Comment:
ideally we should use `with open(...)` statement and let the file be closed on exit

Reply:
Fixed